src/main/java/com/github/alexcojocaru/mojo/elasticsearch/v2/AbstractElasticsearchMojo.java

```diff
@@ -270,14 +272,16 @@ public void setPlugins(ArrayList<PluginConfiguration> plugins)
         this.plugins = plugins;
     }
 
-    public String getPathInitScript()
+    public List<String> getPathInitScripts()
     {
-        return pathInitScript;
+        return Stream.of(pathInitScripts.split(","))
+                .map(String::trim)
+                .collect(Collectors.toList());
```

Comment:
since this is not a dumb getter any longer, it would be nice if you unit test it

Reply:
Done, I test `AbstractElasticsearchMojo` directly.

As I am unsure how exactly will Maven pass the configuration down, I added back the check for not null and not empty in the get method, otherwise the split will fail in the first case (NullPointerException) or return a value in the second case, while in reality we want an empty list in both. You can verify this behaviour in the `AbstractElasticsearchMojoTest` class. If the check is removed, the first two tests will fail.

If you are 100% sure we will never have such a situation, we can then safely remove the superfluous check from the get method